src/bundler.ts

```diff
@@ -46,6 +47,7 @@ export interface Options {
   // or other hint to fix the top-level folder.
   basePath?: string;
 
+  // TODO(usergenic): Added Imports is not yet supported.
```

Comment:
I only want to remove this if it's off the feature list for bundler, since it's planned to bring this forward to maintain parity with vulcanize.